app/views/admin/work_types/edit.html.erb

```diff
@@ -2,14 +2,14 @@
   <h1><span class="fa fa-address-book"></span> <%= t('hyku.admin.work_types') %></h1>
 <% end %>
 
-<div class="panel panel-default">
-  <div class="panel-body">
+<div class="card">
+  <div class="card-body">
     <%= simple_form_for @site, url: '/admin/work_types' do |f| %>
       <% Hyrax.config.registered_curation_concern_types.each do |type| %>
-        <div class="checkbox">
-          <label for="input-<%= type %>">
-            <%= check_box_tag 'available_works[]', type, @site.available_works&.include?(type), id: "input-#{type}" %>
-            <span><%= type %></span><br />
+        <div class="form-check">
+          <input class="form-check-input" type="checkbox" value="<%= type %>" id="input-<%= type %>" name="available_works[]" <%= 'checked' if @site.available_works&.include?(type) %>>
+          <label class="form-check-label" for="input-<%= type %>">
```

Comment:
Reading note: With this change the label moved after the input...curious.

Reply:
i had to look it up but it looks like it could either be before or after or label wrapped around the input, i was just used to putting the label after the input